app/assets/stylesheets/custom.css

```diff
@@ -1389,7 +1389,7 @@ background-color: white;
 
 .stpack {
```

Comment:
@arunkumarsekar06 Is this firefox fix provided by enixel in nilavu code ? 
@megamsys/megam-dev  fyi.